tests/periph_flashpage/main.c

```diff
@@ -366,6 +526,15 @@ static const shell_command_t shell_commands[] = {
     { "test_last", "Write and verify test pattern on last page available", cmd_test_last },
 #ifdef MODULE_PERIPH_FLASHPAGE_RAW
     { "test_last_raw", "Write and verify raw short write on last page available", cmd_test_last_raw },
+#endif
+#ifdef FLASHPAGE_RWWEE_NUMOF
+    { "read_rwwee", "Copy the given page from RWWEE to the local page buffer and dump to STDOUT", cmd_read_rwwee },
+    { "write_rwwee", "Write the local page buffer to the given RWWEE page", cmd_write_rwwee },
+    { "test_rwwee", "Write and verify test pattern to RWWEE", cmd_test_rwwee },
+    { "test_last_rwwee", "Write and verify test pattern on last RWWEE page available", cmd_test_last_rwwee },
+#ifdef MODULE_PERIPH_FLASHPAGE_RAW
```

Comment:
Why adding this #ifdef here while you have the very same a few lines above ?

Reply:
You mean the MODULE_PERIPH_FLASHPAGE_RAW? But this is nested inside FLASHPAGE_RWWEE_NUMOF while the other is not (if I didn't understand wrong what you mean)
(I could nest the other way around, but I kept the same order for rwwee commands as compared to normal ones)